languages/csharp/exercises/concept/chars/.meta/Example.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using System.Text;
+
+public static class CharUtils
+{
+    public static bool DetectInvalidCharComination(string str)
+    {
+        const char DASH = '-';
+        const char GREATER_THAN = '>';
+        for (int i = 1; i < str.Length - 1; i++)
+        {
+            if (str[i] == DASH && str[i + 1] == GREATER_THAN && str[i - 1] != DASH)
+            {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    public static string ToUpper(string str)
```

Comment:
Couldn't this be solved by just doing `public static string ToUpper(string str) => str.ToUpper();`